Gemfile.lock

```diff
@@ -83,30 +83,31 @@ GEM
       rake (>= 10.4, < 14.0)
     apipie-rails (0.5.18)
       rails (>= 4.1)
-    arbre (1.2.1)
-      activesupport (>= 3.0.0)
-    ast (2.4.0)
+    arbre (1.3.0)
```

Comment:
### [arbre](https://github.com/activeadmin/arbre)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.2.1 → 1.3.0

[_~~change-log~~, [source-code](https://github.com/activeadmin/arbre)_]





<details>
<summary>Commits</summary>

A change of **121** commits. See the full changes on [the compare page](https://github.com/activeadmin/arbre/compare/v1.2.1...v1.3.0).

These are the first 10 commits:
- (d77bfa7) [Bump capybara from 3.15.0 to 3.15.1 (#123)](https://github.com/activeadmin/arbre/commit/d77bfa71d78bf9515e02b77171f9acdb5e69544e)
- (4081d70) [[Security] Bump nokogiri from 1.10.2 to 1.10.3 (#124)](https://github.com/activeadmin/arbre/commit/4081d70c3a33edef3ba86c2a47feaba2b30f4e71)
- (b7a553e) [Bump rubocop from 0.67.2 to 0.68.0 (#125)](https://github.com/activeadmin/arbre/commit/b7a553e5f265be74dea443bea9196407259fe9c1)
- (5abe62d) [Bump rubocop from 0.68.0 to 0.68.1](https://github.com/activeadmin/arbre/commit/5abe62d69e823c49ae6ce8056d6ecde50d612967)
- (8621789) [Merge pull request #126 from activeadmin/dependabot/bundler/…](https://github.com/activeadmin/arbre/commit/8621789d2f3fcc8b92ca11cdded6383bba6fcf55)
- (a46af6c) [Bump rubocop from 0.68.1 to 0.69.0 (#127)](https://github.com/activeadmin/arbre/commit/a46af6c43c976224d2e4611ad02941eab5eb493b)
- (86d89d2) [Bump rubocop from 0.69.0 to 0.70.0](https://github.com/activeadmin/arbre/commit/86d89d24e11c542222c24593547a67daf8030c1b)
- (b1dda76) [Merge pull request #128 from activeadmin/dependabot/bundler/…](https://github.com/activeadmin/arbre/commit/b1dda76bdd31a84f5f611025df5523df1a108c9b)
- (6e6c2bf) [Bump rubocop from 0.70.0 to 0.71.0](https://github.com/activeadmin/arbre/commit/6e6c2bfda8ece6658958f31a8b8ef04db84421e2)
- (2253669) [Merge pull request #131 from activeadmin/dependabot/bundler/…](https://github.com/activeadmin/arbre/commit/22536691412784e540e54e8d0ebd0070edc8b8a3)


</details>